doc/doc-site/docs/overview/orders.md

```diff
@@ -592,14 +563,19 @@ Hence decimal tick size must be reduced so that integer prices can fit into a 32
 As shown above, picking the "correct" lot size, tick size, and minimum order size for a given market is ultimately a judgement call.
 Still, however, there are several guidelines that can aid the process:
 
+1. **One tick should be small in value relative to one lot**:
+   since the minimum expressible price is 1 tick per lot, followed by 2, 3 and so on, it's ideal for these prices to be small relative to the value of one lot.
+
 1. **Only specify as much granularity as needed**:
    overly-granular lot size and tick size combinations do not properly translate to integer tick sizes, and may make indexing more difficult.
    Three orders at price 12.34 is easier to interpret than one order each at 12.3401, 12.3404, and 12.3407.
+   In addition, note that lowering the tick size also decreases the maximum expressible price (always worth $2^{32}-1$ ticks).
+   There is a Python helper, among those above, for calculating the maximum price in decimal unit terms: `get_max_quote_per_base_nominal` (see `get_min_quote_per_base_nominal`).
```

Comment:
I suggest including examples of these helpers above:

* `get_min_quote_per_base_nominal` for the first example under "Units and market parameters"
* `get_max_quote_per_base_nominal` under the wBTC examples

Then in this section probably reference how the SDK functions are used in the walkthrough and talk about playing around with them more

Ideally this will link to the Python SDK page with a relative doc link (something like ../off-chain/python-sdk)